packages/flutter_tools/templates/app/linux.tmpl/main.cc

```diff
@@ -1,9 +1,8 @@
 #include "my_application.h"
 
 int main(int argc, char** argv) {
-  // Only X11 is currently supported.
-  // Wayland support is being developed: https://github.com/flutter/flutter/issues/57932.
-  gdk_set_allowed_backends("x11");
+  // We only currently support X11 and Wayland.
```

Comment:
s/We/The engine/, since it's not at all clear in the context of code that's part of developers' projects who "we" would be referring to.

I think we should probably drop "currently" for the same reason. It's actually the list of things that were supported by the engine at the time that the person ran `flutter create`.

Reply:
Perhaps `// The Flutter Linux platform supports both X11 and Wayland`.

All the GTK platforms we are excluding are for other OSs. There are no and will be no other Linux platforms supported by GTK3.

Reply:
Should we just remove the `gdk_set_allowed_backends` call entirely then?

Reply:
I kept the call just in case another backend appeared that we'd never added support for, but yeah, I think in all real cases this wont occur so lets just drop the call.